util/json.h

```diff
@@ -16,14 +16,14 @@
 	json_object_object_add(o, k, json_object_new_int(v))
 #ifdef CONFIG_JSONC_14
 #define json_object_add_value_uint64(o, k, v) \
-	json_object_object_add(o, k, json_object_new_uint64(v))
+	json_object_object_add(o, k, json_object_new_int64(v))
```

Comment:
This doesn't make sense at all. Why should we use the int64 version of the function?